src/safeds/data/tabular/containers/_time_series.py

```diff
@@ -5,10 +5,13 @@
 from typing import TYPE_CHECKING
 
 import matplotlib.pyplot as plt
+import numpy as np
 import pandas as pd
 import seaborn as sns
+import torch
+import xxhash
+from torch.utils.data import DataLoader, Dataset
```

Comment:
In the `src` folder, imports of external libraries must now be moved into the functions that use them (see #624).